src/loaders/compute_tools/eggnog/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM continuumio/miniconda3:24.1.2-0
+
+ENV EGGNOG_VER 2.1.12
+ENV CONDA_ENV eggnog-$EGGNOG_VER
+ENV PYTHON_VER 3.11
+
+RUN conda config --add channels bioconda
+RUN conda config --add channels conda-forge
+
+RUN conda create -n $CONDA_ENV python=$PYTHON_VER
+RUN conda install -n $CONDA_ENV -c conda-forge -c bioconda eggnog-mapper=$EGGNOG_VER
+RUN conda install -n $CONDA_ENV pandas=2.2.1 jsonlines=2.0.0
+
+RUN echo "source activate $CONDA_ENV" >> ~/.bashrc
+
+# eggNOG annotation DB is pre-downloaded at /global/cfs/cdirs/kbase/collections/libraries/eggnog/5.0.2
+# following instructions at https://github.com/eggnogdb/eggnog-mapper/wiki/eggNOG-mapper-v2.1.5-to-v2.1.12#setup
+# Mount the annotation DB directory to /reference_data when running the container
```

Comment:
Can you remind me how much of the DB management / mounting / etc. is manual and how much is automated

Reply:
We need to manually download the DB to collections/libraies/<tool_name>/<db_version>.
If in the versions.yaml file a `reference_db_version` field is filled, the code will automatically mount the db to /reference_data directory.

Reply:
:+1: